src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -8,13 +8,20 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
 {
     internal class LinuxAppServiceEventGenerator : LinuxEventGenerator
     {
+        private readonly IEnvironment _environment;
+        private readonly Action<string> _writeEvent;
         private readonly LinuxAppServiceFileLoggerFactory _loggerFactory;
         private readonly HostNameProvider _hostNameProvider;
+        private string _tenantId;
```

Comment:
Why are these fields needed?

Reply:
If those fields aren't populated, they're not. I think I was hewing a little too closely to what was being done in LinuxContainerEventGenerator. I'll remove them.